src/core/Env.js

```diff
@@ -76,7 +76,7 @@ getJasmineRequireObj().Env = function(j$) {
        * @type Boolean
        * @default false
        */
-      hideDisabled: false
+      hideDisabled: false,
```

Comment:
This, and below in `configure` look like they're left over from the previous version with the feature being configurable. Let's pull them out so as not to confuse.

Reply:
done